.eslintrc

```diff
@@ -7,14 +7,15 @@
     "browser": true
   },
   "rules": {
-    "react/jsx-props-no-spreading": { "custom": "ignore" },
-    "react/prop-types": "off",
-    "no-nested-ternary": "off",
+    "react/jsx-props-no-spreading": 0, // disables prop spreading messages in src/components/link.jsx
+    "react/prop-types": 0, // disables prop type messages
+    "no-nested-ternary": "off", // disables nested ternary messages
```

Comment:
This rule allows for nested ternaries. In general, I think phrasing these comments as "disabling messages" is slightly different

Reply:
Replaced with `"no-nested-ternary": "off", // allows nested ternaries`

Reply:
Thanks! Should the other comments be changed too?

Reply:
It is currently
```
"react/jsx-props-no-spreading": "off", // allows prop spreading (used in src/components/link.jsx)
"react/prop-types": "off", // disables prop type messages
"no-nested-ternary": "off", // allows nested ternaries
```

Reply:
Okay I think the second one is something more along the lines of `does not require props validation in a React component definition`

Reply:
Updated!